File: HouseM8API/HouseM8API/Models/Users/MateUpdate.cs
using HouseM8API.Entities;

namespace HouseM8API.Models.Users
{
    public class MateUpdate
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string UserName { get; set; }
        public string Description { get; set; }
        public Address Address { get; set; }
        public int Range { get; set; }
    }
}

File: HouseM8-Flutter/housem8_flutter/lib/models/mate_update.dart
import 'package:housem8_flutter/models/address.dart';

class MateUpdate {
  String firstName;
  String lastName;
  String userName;
  String description;
  Address address;
  int range;

  MateUpdate(
      {this.firstName,
      this.lastName,
      this.userName,
      this.description,
      this.address,
      this.range});

  factory MateUpdate.fromJson(Map<String, dynamic> json) {
    return MateUpdate(
      firstName: json["firstName"],
      lastName: json["lastName"],
      userName: json["userName"],
      description: json["description"],
      address: Address.fromJson(json["address"]),
      range: json["range"],
    );
  }
}
